client/lib/products-values/index.js

```diff
@@ -321,5 +329,6 @@ module.exports = {
 	isUnlimitedSpace,
 	isUnlimitedThemes,
 	isVideoPress,
-	whitelistAttributes
+	whitelistAttributes,
+	isEmailVerified
```

Comment:
We should try to keep exports ordered alphabetically. It's much more easier to scan that way.